common/overlay/overlay.go

```diff
@@ -16,12 +16,17 @@ package overlay
 
 import (
 	"fmt"
+	"strings"
 	"syscall"
 
 	"github.com/coreos/rkt/pkg/label"
 	"github.com/hashicorp/errwrap"
 )
 
+// sanitizer defines a string translator used to escape colon and coma
+// characters in the directories names.
+var sanitizer = strings.NewReplacer(":", "\\:", ",", "\\,")
```

Comment:
Sorry for not spotting this earlier, but if I understand correctly the `\\` (double-slash) in here is just for golang string escaping right? To make a bit clearer that we are doing the escaping for overlayfs here, would you mind changing this into raw strings with a single slash?